formidable/views.py

```diff
@@ -21,6 +24,86 @@
 from formidable.serializers.presets import PresetsSerializer
 
 
+logger = logging.getLogger(__name__)
+
+
+def extract_function(func_name):
```

Comment:
The good method can be : https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/settings.py#L169